sdk/adp/Azure.AI.AutonomousDevelopmentPlatform/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0002:DO ensure all service methods, both asynchronous and synchronous, take an optional CancellationToken parameter called cancellationToken.", Justification = "CancellationToken can be passed through RequestOptions")]
```

Comment:
FYI https://github.com/Azure/azure-sdk-tools/issues/4715

Reply:
@m-nash I think the global suppression here is unnecessary. I've posted details in the issue above.